nova/virt/vmwareapi/driver.py

```diff
@@ -336,6 +337,7 @@ def get_available_resource(self, nodename):
 
         """
         host_stats = self._vc_state.get_host_stats(refresh=True)
+        LOG.debug("HOST STATS: %s" % host_stats)
```

Comment:
Could you remove this debug output line? Otherwise, it looks good to go for me.
Thanks